examples/02_decoding/plot_oasis_vbm.py

```diff
@@ -168,29 +183,39 @@
 
 # Statistical inference
 from nilearn.mass_univariate import permuted_ols
+
 neg_log_pvals, t_scores_original_data, _ = permuted_ols(
-    age, data,  # + intercept as a covariate by default
+    age,
+    data,  # + intercept as a covariate by default
     n_perm=2000,  # 1,000 in the interest of time; 10000 would be better
-    verbose=1, # display progress bar
-    n_jobs=1)  # can be changed to use more CPUs
+    verbose=1,  # display progress bar
+    n_jobs=1,
+)  # can be changed to use more CPUs
```

Comment:
If we think this comment is really useful, then we should move it to the line just above now